srcpkgs/python3-resolvelib/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'python3-resolvelib'
+pkgname=python3-resolvelib
+version=0.7.1
+revision=1
+wrksrc="${pkgname/python3-/}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
```

Comment:
This should have `depends="python3"` since it does not depend on any other python package. Also please try adding `checkdepends="python3-pytest"` to see if we can get tests to run successfully.

Reply:
it has more unpackaged test dependencies, do we need to block this on that? I can package them as well, but I'm not sure when I'd get to that.

Reply:
Done